server/__tests__/unit/campaigns.test.js

```diff
@@ -0,0 +1,29 @@
+// server/__tests__/api/campaigns.test.js
+const request = require('supertest')
+const app = require('../../app')
+const Campaign = require('../../db/models/campaign')
+
+jest.mock('../../db/models/campaign', () => {
+  return {
+    query: jest.fn().mockReturnThis(),
+    findById: jest.fn().mockResolvedValue({ id: 1, name: 'Campaign 1' }),
+    deleteById: jest.fn().mockResolvedValue(true)
+  }
+})
+
+describe('DELETE /api/campaigns/:id', () => {
```

Comment:
Since you have to move the delete endpoint to /api/v1/campaigns as mentioned in a different comment, don't forget to update the tests!